{
  "uid" : "95b0799d9cb71618",
  "name" : "test_xfail_expected_failure",
  "fullName" : "vkr_project_tests.test_attach#test_xfail_expected_failure",
  "historyId" : "a55201de92fab6edbf92cb844607333f",
  "time" : {
    "start" : 1649368400357,
    "stop" : 1649368400358,
    "duration" : 1
  },
  "description" : "this test is an xfail that will be marked as expected failure",
  "descriptionHtml" : "<p>this test is an xfail that will be marked as expected failure</p>\n",
  "status" : "skipped",
  "statusMessage" : "XFAIL this test is expecting failure\n\nAssertionError: assert False",
  "statusTrace" : "@pytest.mark.xfail(condition=lambda: True, reason='this test is expecting failure')\n    def test_xfail_expected_failure():\n        \"\"\"this test is an xfail that will be marked as expected failure\"\"\"\n>       assert False\nE       assert False\n\nvkr_project_tests/test_attach.py:7: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "tag",
    "value" : "@pytest.mark.xfail(condition=<function <lambda> at 0x7f03fde843a0>, reason='this test is expecting failure')"
  }, {
    "name" : "parentSuite",
    "value" : "vkr_project_tests"
  }, {
    "name" : "suite",
    "value" : "test_attach"
  }, {
    "name" : "host",
    "value" : "fv-az121-167"
  }, {
    "name" : "thread",
    "value" : "2313-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "vkr_project_tests.test_attach"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ ],
    "history" : {
      "statistic" : {
        "failed" : 0,
        "broken" : 0,
        "skipped" : 4,
        "passed" : 0,
        "unknown" : 0,
        "total" : 4
      },
      "items" : [ {
        "uid" : "8f964f545b220d7e",
        "reportUrl" : "https://YaroslavS1.github.io/VKR_project/38//#testresult/8f964f545b220d7e",
        "status" : "skipped",
        "statusDetails" : "XFAIL this test is expecting failure\n\nAssertionError: assert False",
        "time" : {
          "start" : 1649367904473,
          "stop" : 1649367904474,
          "duration" : 1
        }
      }, {
        "uid" : "df0c2a5d6391ed56",
        "reportUrl" : "https://YaroslavS1.github.io/VKR_project/36//#testresult/df0c2a5d6391ed56",
        "status" : "skipped",
        "statusDetails" : "XFAIL this test is expecting failure\n\nAssertionError: assert False",
        "time" : {
          "start" : 1649366491603,
          "stop" : 1649366491603,
          "duration" : 0
        }
      }, {
        "uid" : "113423405c395fed",
        "reportUrl" : "https://YaroslavS1.github.io/VKR_project/35//#testresult/113423405c395fed",
        "status" : "skipped",
        "statusDetails" : "XFAIL this test is expecting failure\n\nAssertionError: assert False",
        "time" : {
          "start" : 1649366098401,
          "stop" : 1649366098401,
          "duration" : 0
        }
      } ]
    },
    "tags" : [ "@pytest.mark.xfail(condition=<function <lambda> at 0x7f03fde843a0>, reason='this test is expecting failure')" ]
  },
  "source" : "95b0799d9cb71618.json",
  "parameterValues" : [ ]
}